packages/package_info/example/macos/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Flutter-related
```

Comment:
Do we need this file?

Reply:
It's part of the template. Having a monolithic .gitignore for the whole project doesn't scale well to N (optional) platforms, so all platform-specific ignores now live in their respective platform directories.

Reply:
SG thx